ordered_product/models.py

```diff
@@ -0,0 +1,101 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+from django.core.validators import MinValueValidator
+from django.core.exceptions import ValidationError
+from product.models import Product
+
+
+class User(models.Model):
```

Comment:
Are all the classes in this file are temporary? because we don't need to redefine them.
if so, please mention it in the description.